### PR TITLE
Add needed files for MCXW71 BLE enablement

### PR DESCRIPTION
**Pull request by yeaissa**
Adding needed files to enable BLE apps on MCXW71

### CODE REVIEW

File: mcux/middleware/mcux-sdk-middleware-connectivity-framework/platform/connected_mcu/fwk_platform.c
Comment (danieldegrasse) [inline mcux/middleware/mcux-sdk-middleware-connectivity-framework/platform/connected_mcu/fwk_platform.c:1]: Can you shorten commit message and add commit body?

I'm not aware of a specific restriction requiring us to follow Zephyr commit guidelines in the HAL but it seems like good practice to keep things consistent

File: mcux/hal_nxp.cmake
Comment (danieldegrasse) [inline mcux/hal_nxp.cmake:1]: Same comment here- I think we should add a commit body

File: mcux/README [hunk 1]
```diff
@@ -17,7 +17,15 @@ Origin:
 
    NXP MCUXpresso SDK Connectivity Framework
    https://github.com/nxp-mcuxpresso/mcux-sdk-middleware-connectivity-framework
-   TAG: MCUX_2.15.000_RW610_RC1
+   TAG: MCUX_2.15.000_RW610_RC1 - MCUX_2.16.000(For connected_mcu platform files used by MCXW71)
```
Comment (danieldegrasse) [inline mcux/README:20]: These changes should be placed with the commits that actually import the middlewares

File: mcux/README [hunk 2]
```diff
@@ -88,6 +96,7 @@ Patch List:
   - mcux-sdk-middleware-connectivity-framework: Add new CMake file connectivity_framework.cmake
   - mcux-sdk-middleware-connectivity-framework: rw61x: Fix assertion OSA Mutex Alloc fail
   - mcux-sdk-middleware-connectivity-framework: rw61x: Remove static from hci_cal_data buffers to be used inside Zephyr
+  - mcux-sdk-middleware-connectivity-framework: connected_mcu: Modified to remove a large amount of dependencies and adapt code to be supported with Zephyr
```
Comment (danieldegrasse) [inline mcux/README:99]: Ditto here- place these changes with the commit importing the middleware

**Comment by yeaissa** [comment]
Updated, Thanks for the review @danieldegrasse. Please check again

**Review by danieldegrasse — APPROVED** [review]

**Review by dleach02 — APPROVED** [review]

**Pull request merged**